src/Raven.Server/Storage/Schema/Updates/Server/52101/From50000.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Binary;
+using Sparrow.Json;
+using Voron;
+using Voron.Data.Tables;
+using Voron.Impl;
+
+namespace Raven.Server.Storage.Schema.Updates.Server
+{
+    internal class From50000 : ISchemaUpdate
```

Comment:
Can we have a test that will verify this schema upgrade?

Reply:
This will be handled later as part of testing.